2009-03-18	Nicolas Setton  <setton@adacore.com>

	* configure: Under darwin versions > 9.4.0, xrandr.pc describes
       	 version 1.2.2, but the Xrandr.h shipped with the system does not
         define symbols used by Gtk+, so we disable the use of RANDR.

--- configure.original	2009-03-19 17:55:51.000000000 +0100
+++ configure	2009-03-19 17:56:20.000000000 +0100
@@ -33743,7 +33743,7 @@
   if $PKG_CONFIG --exists "xrandr >= 1.2" ; then
 
 cat >>confdefs.h <<\_ACEOF
-#define HAVE_RANDR 1
+#undef HAVE_RANDR
 _ACEOF
 
 
